src/cr/cube/dimension.py

```diff
@@ -106,7 +106,8 @@ def hs_indices(self):
                 if el['id'] == subtotal.anchor
             ][0]
             if (subtotal.anchor in [el['id'] for el in self._elements]) else
-            -1,
+            subtotal.anchor,
+            # -1,
```

Comment:
I am a big fan of Python's list comprehension, but this is incredibly difficult to read, is there a way to refactor this, potentially into multiple statements to make it more clear?